File: Library/WindowsApplication1/WindowsApplication1/Form1.vb
Public Class Form1
    Dim xlApp

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        xlApp = CreateObject("SerialPortDataProcessor.PrintLabelProcessor")
        xlApp.Init()
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        xlApp = CreateObject("SerialPortDataProcessor.PrintLabelProcessor")
        xlApp.Init()
        xlApp.PrintLabel(100)
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        xlApp.StopReading()
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Dim res
        res = xlApp.ReadedData()
        TextBox1.Text = res
    End Sub
End Class

File: Library/WindowsApplication1/WindowsFormsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using SerialPortDataProcessor;

namespace WindowsFormsApplication1
{
    public partial class Form1 : Form
    {
        private SerialPortDataProcessor.ComPortDataProcessor _portDataProcessor;
        private SerialPortDataProcessor.PrintLabelProcessor _printLabelProcessor = new PrintLabelProcessor();

        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            _portDataProcessor.StartReading();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            _portDataProcessor = new ComPortDataProcessor();
            _portDataProcessor.InitReading("COM1", 8, 9600);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            _portDataProcessor.StopReading();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            MessageBox.Show(_portDataProcessor.ReadedData());
        }

        private void button4_Click(object sender, EventArgs e)
        {
            _printLabelProcessor.Init();
            _printLabelProcessor.PrintLabelV2(100);
        }
    }
}
